letterparser/parse.py

```diff
@@ -1,4 +1,38 @@
-import pypandoc
+import os
+import docker
+from letterparser.utils import unicode_encode, unicode_decode
 
-def dummy():
-    return True
+
+DOCKER_IMAGE = "knsit/pandoc:v2.7"
+
+
+def get_docker_client():
+    return docker.from_env()
+
+
+def get_docker_volume_and_file(file_name):
```

Comment:
I would recommend turning this into two functions. One to get the volume and the other the file (see single responsibility principle).